src/main/g8/build.sbt

```diff
@@ -56,7 +56,8 @@ lazy val root: Project = project
         ExclusionRule("com.twitter", "chill_2.11"),
         ExclusionRule("org.apache.flink", "flink-clients_2.11"),
         ExclusionRule("org.apache.flink", "flink-runtime_2.11"),
-        ExclusionRule("org.apache.flink", "flink-streaming-java_2.11")
+        ExclusionRule("org.apache.flink", "flink-streaming-java_2.11"),
+        ExclusionRule("org.apache.flink", "flink-optimizer_2.11")
```

Comment:
Actually since we are excluding it we need to add back the right dep. I'll fix it

Reply:
Ah makes sense.  Thank you!